ci/bamboo/build-manylinux-prototype.sh

```diff
@@ -1,101 +0,0 @@
-#!/bin/bash
```

Comment:
i.e. `build_manylinux_wheel.sh` should replace `build-linux-release.sh` and `build-linux-debug.sh` and remain in `ci/bamboo/`.